setup.py

```diff
@@ -23,7 +23,7 @@ def load_req(filename):
 
 
 setup(name='tokenserver',
-      version='1.5.3',
```

Comment:
I seem to have tags for 1.5.4 and 1.5.5 so I guess maybe we missed incrementing this here?

Reply:
I think jr may have tried to make some releases but failed because python 2 is so old now it's hard to get dependency versions that work with it. (Never thought I'd see the day!)